drivers/adc/Kconfig.ad4130

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2025 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config ADC_AD4130
+	bool "ADI AD4130 Analog-to-Digital Converter"
+	default y
+	depends on DT_HAS_ADI_AD4130_ADC_ENABLED
+	select SPI
+	select ADC_CONFIGURABLE_INPUTS
+	help
+	  Enable the AD4130 ADC driver. It is SPI based
+	  Ultra Low Power, 24-Bit Sigma-Delta ADC.
+
+config ADI_AD4130_ADC_ACQUISITION_THREAD_INIT_PRIO
```

Comment:
Should just be PRIO, INIT_PRIO usually refers to initialization priority :)

Reply:
Thanks, done.